command/rootchain/emit/emit.go

```diff
@@ -59,6 +59,13 @@ func setFlags(cmd *cobra.Command) {
 		nil,
 		"list of amounts to fund wallets",
 	)
+
+	cmd.Flags().StringVar(
+		&params.jsonRPCAddress,
+		jsonRPCFlag,
+		"",
```

Comment:
If it is required for the command, default to `localhost:8545`.

Reply:
It is required, but it will fallback to resolving docker IP address. But I can default it to localhost if that makes more sense...?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/989/commits/3732f6e0b662b6ace6f9c297999c9ff6ffc5efd1

A note there:
I've tried supplying `localhost:8545`, but seems `ethgo` cannot handle it since it expects specifying protocol, IP address and port notation instead.